src/django/api/processing.py

```diff
@@ -492,3 +494,72 @@ def make_pending_match(item_id, facility_id, score):
         update_extendedfields_for_list_item(item)
 
     return all_matches
+
+
+def save_exact_match_details(exact_results):
+    """
+    Save the results of a call to identify_exact_matches by creating
+    Facility and FacilityMatch instances and updating the state of the affected
+    FacilityListItems.
+
+    Should be called in a transaction to ensure that all the updates are
+    applied atomically.
+
+    Arguments:
+    exact_results -- The dict return value from a call to
+                    identify_exact_matches
+
+    Returns:
+    The list of `FacilityMatch` objects created
+    """
+    processed_list_item_ids = exact_results['processed_list_item_ids']
+    item_matches = exact_results['item_matches']
+    started = exact_results['started']
+    finished = exact_results['finished']
+
+    def make_pending_match(item_id, facility_id):
+        return FacilityMatch(
+            facility_list_item_id=item_id,
+            facility_id=facility_id,
+            confidence=1.0,
+            status=FacilityMatch.PENDING,
+            results={})
+
+    all_matches = []
+    for item_id, exact_matches in item_matches.items():
+        item = FacilityListItem.objects.get(id=item_id)
+        item.status = FacilityListItem.POTENTIAL_MATCH
+
+        matches = [make_pending_match(item_id, m.get('facility_id'))
+                   for m in exact_matches]
```

Comment:
After running through the testing instructions, the last list item ended up with 3 match rows. I originally thought this was a problem, but I confirmed that this matches the behavior of the normal deduple process where we would create matches for both a 60% and 90% confidence match but the high quality match would be the `AUTOMATIC` winner. 👍 

```
openapparelregistry=# select id, results, status, facility_id, facility_list_item_id from api_facilitymatch where facility_list_item_id = 935;
 id  |                 results                  |  status   |   facility_id   | facility_list_item_id
-----+------------------------------------------+-----------+-----------------+-----------------------
 937 | {"match_type": "multiple_exact_matches"} | AUTOMATIC | BD202200698XDZC |                   935
 938 | {}                                       | PENDING   | US202200661FFPF |                   935
 939 | {}                                       | PENDING   | US202200661FFPF |                   935
(3 rows)
```